### PR TITLE
Added a bunch of packages that may be useful for everyone

### PR DESCRIPTION
**Pull request by qbunt**

### CODE REVIEW

File: adium.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   tasks:
     - name: install Adium
       mac_pkg: pkg_type=app
-               url=http://sourceforge.net/projects/adium/files/Adium_1.5.7.dmg/download
+               url=http://downloads.sourceforge.net/project/adium/Adium_1.5.7.dmg?r=&ts=1424418546&use_mirror=iweb
```
Comment (spencergibb) [inline adium.yml]: This won't work with a timestamp, granted the one it's replacing doesn't work either.

File: osx.yml [hunk 2]
```diff
@@ -0,0 +1,176 @@
+---
+- hosts: workstation
+
+  tasks:
```
Comment (spencergibb) [inline osx.yml]: did you mean to include this file?  This should be personal, I think.

Comment (qbunt) [inline osx.yml]: I did mean to leave that one out, sorry about that.

**Comment by qbunt** [comment]
Made some changes based on your comments, that Adium download is annoying, maybe another non-sourceforge mirror exists that I'm unaware of.

**Comment by qbunt** [comment]
Thanks man!

**Pull request merged**